internal/cli/config/describe.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/spf13/cobra"
+)
+
+type ListOpts struct {
+	name string
+}
+
+func (opts *ListOpts) Run() error {
+	configDescription := config.GetConfigDescription(opts.name)
+
+	if len(configDescription) == 0 {
+		fmt.Printf("There is no config with name '%v'\n", opts.name)
```

Comment:
[non-blocking] 
```suggestion
		return fmt.Errorf("no profile with name '%s'\n", opts.name)
```
(errors should not start with caps)